pyicloud/services/photos.py

```diff
@@ -213,8 +214,9 @@ def size(self):
 
     @property
     def created(self):
-        dt = datetime.fromtimestamp(self.data.get('createdDate') / 1000.0)
-        return dt.strftime('%Y-%m-%dT%H:%M:%SZ')
+        dt = datetime.utcfromtimestamp(self.data.get('createdDate') / 1000.0)
+        dt = dt.replace(tzinfo=pytz.utc)
```

Comment:
Out of curiosity, why is this needed? `tzinfo=None` seems to indicate UTC by default?